pkg/option/config.go

```diff
@@ -163,6 +163,11 @@ const (
 	// in lb{4,6}_backend in BPF map.
 	FixedZoneMapping = "fixed-zone-mapping"
 
+	// EnableActiveConnectionTracking enables counting open and active
+	// connections to services, grouped by zones defined in
+	// FixedZoneMapping.
+	EnableActiveConnectionTracking = "enable-active-connection-tracking"
```

Comment:
*DaemonConfig is the legacy way of defining config flags, so its best to avoid adding new flags to it if possible.
 
Related to the previous comment for using cell.Cell to expose the new lrs map, [we could use `cell.Config` to parse ](https://github.com/cilium/cilium/blob/5b1dce19047f35effa2dae3a270ad1e139f10479/pkg/maps/nodemap/cell.go#L26-L36)this configuration inside pkg/maps/lrs.